pom.xml

```diff
@@ -73,7 +73,9 @@
 
 		<spring-cloud-skipper.version>2.2.0.BUILD-SNAPSHOT</spring-cloud-skipper.version>
 
-		<spring-cloud-task.version>2.1.3.BUILD-SNAPSHOT</spring-cloud-task.version>
+		<spring-cloud-task.version>2.2.0.BUILD-SNAPSHOT</spring-cloud-task.version>
+
+		<spring-security-oauth-autoconfigure.version>2.2.0.M4</spring-security-oauth-autoconfigure.version>
```

Comment:
We can use the name of the property to be `spring-security-oauth2-autoconfigure.version`  (oauth -> oauth2) based on the convention of using the artifact name. Will fix this.